datalad/interface/common_opts.py

```diff
@@ -151,7 +151,9 @@
     w/o git-annex being aware of it. In case of a change in origin you need to
     update the clone before you're able to save new content on your end.
     Alternative to 'auto' when hardlinks are not an option, or number of consumed
-    inodes needs to be minimized.
+    inodes needs to be minimized. Please note, that you might run into issues
+    when using this mode to link a bare (or any tuned annex) and a non-bare 
+    repository that way, since their annex object trees are organized differently.
```

Comment:
Maybe: Using this mode with bare repositories (outside of a RIA store) or tuned repositories is not supported, because their annex object trees are organized  differently.

It will not work. There is no uncertainty

Reply:
Don't think so. First of all: It is supported in that `--reckless=ephemeral` will do it's thing. Second, there is uncertainty, since you can have a tuned repo, using dirhashlower. It will work in that case. Third: At some point I tested, that a bare repo can actually deal with dirhashmixed keys in its object tree. It just wouldn't put it in there itself. So, if you use an ephemeral clone to write to it you can also read those keys from another clone. But the linked ephemeral clone can't handle existing dirhashlower objects in the bare repo.
All depends on the setup. Can work or not.

Reply:
But will add "outside RIA store" as a hint.